propagation-stackdriver/README.md

```diff
@@ -0,0 +1,28 @@
+# propagation-stackdriver
+
+This is a propagation that behaves like `B3Propagation`, but also extracts the tracing context from the 'x-cloud-trace-context' key.
```

Comment:
only thing I'd suggest is a code block here showing putting this in your build as from the intro sentence it isn't clear which type people should use and where. so.. make that the next thing and sounds good!

Reply:
sorry I meant to say "builder" ex Tracer.Builder.propagationFactory(what goes here)